.github/workflows/publish-docker.yml

```
@@ -4,7 +4,7 @@ on:
   push:
     branches: [master, release/*]
     paths:
-      - 'src/Nethermind/**'
+      - "src/Nethermind/**"
```

Double quotes again :)

```suggestion
      - 'src/Nethermind/**'
```